agents/host-amd64/agent-config.yaml

```diff
@@ -1,7 +1,3 @@
-server:
-  # log_level: debug
-  http_listen_port: 12345
-
```

Comment:
Even here, we may want to look at the newer config that's packaged with the docker-compose setup